torch/csrc/jit/mobile/model_tracer/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+set(
+  MODEL_TRACER_DIR
+  "${TORCH_ROOT}/torch/csrc/jit/mobile/model_tracer")
+set(MODEL_TRACER_DIR
+  ${TORCH_ROOT}/torch/csrc/jit/mobile/model_tracer/tracer.cpp
+  ${TORCH_ROOT}/torch/csrc/jit/mobile/model_tracer/TensorUtils.cpp
+  ${TORCH_ROOT}/torch/csrc/jit/mobile/model_tracer/MobileModelRunner.cpp
+  ${TORCH_ROOT}/torch/csrc/jit/mobile/model_tracer/OperatorCallTracer.cpp
+  ${TORCH_ROOT}/torch/csrc/jit/mobile/model_tracer/KernelDTypeTracer.cpp
+)
```

Comment:
Please avoid filelists in CMakeFiles, use `build_variables.bzl` instead

Reply:
Good point! Address in the latest commit.